include/jwt-cpp/traits/boost-json/traits.h

```diff
@@ -67,7 +67,8 @@ namespace jwt {
 				return val.get_double();
 			}
 
-			static bool parse(value_type& val, string_type str) {
+			template<class string_t>
```

Comment:
should this not be specialized based on what the library supports?

Reply:
`json::parse` accepts a templated type. IMHO it's safe to also take any string type.